packages/grid/x-data-grid-premium/src/components/GridExcelExportMenuItem.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import MenuItem from '@mui/material/MenuItem';
+import { GridExportMenuItemProps } from '@mui/x-data-grid';
```

Comment:
I tend to import every from `@mui/x-data-grid-pro` in `@mui/x-data-grid-premium` (same for `@mui/x-data-grid-pro/internals`

It has two benefits

1. Double check that we do export everything of the Community Plan inside the Pro Plan
2. If you have to import elements from both Community and Pro Plan, you can do it in a single line